File: android/app/src/main/java/com/example/dobpicker/utils/DateUtils.kt
package com.example.dobpicker.utils

import java.text.ParseException
import java.text.SimpleDateFormat
import java.util.Date
import java.util.Locale

object DateUtils {

    private const val DATE_FORMAT = "yyyy-MM-dd"
    val dateFormatter = SimpleDateFormat(DATE_FORMAT, Locale.US)

    fun parseDate(strDate: String): Date? {

        var date: Date? = null
        try {
            date = dateFormatter.parse(strDate)
        } catch (e: ParseException) {
            e.printStackTrace()
        }
        return date
    }
}

File: ios/App/App/DateUtils.swift
import Foundation

class DateUtils {

    private static let dateFormat = "yyyy-MM-dd"

    private static func getDateFormatter() -> DateFormatter {
        let dateFormatter = DateFormatter()
        dateFormatter.dateFormat = dateFormat
        return dateFormatter
    }

    static func parseDate(_ strDate: String) -> Date? {
        return getDateFormatter().date(from: strDate)
    }

    static func parseDate(_ date: Date) -> String {
        return getDateFormatter().string(from: date)
    }
}
